src/features/apps/apiSlice.ts

```diff
@@ -149,11 +149,17 @@ export const apiSlice = createApi({
       query: () => '/api/apps/subscribed/activesubscriptions',
     }),
     fetchSubscriptionApp: builder.query<
-      ActiveSubscriptionDetails,
+      FetchSubscriptionResponseType,
       FetchSubscriptionAppQueryType
     >({
       query: (obj) =>
         `/api/apps/${obj.appId}/subscription/${obj.subscriptionId}/subscriber`,
+      transformErrorResponse: (res) => {
+        return {
+          status: res.status,
+          data: res.data,
+        }
+      },
```

Comment:
return JSON more compact. instead of
`(res) => { return {...} }`
write
`(res) => ({...}))`